web/core/components/workspace-notifications/sidebar/header/options/root.tsx

```
@@ -1,6 +1,7 @@
 import { FC } from "react";
 import { observer } from "mobx-react";
 import { CheckCheck, RefreshCw } from "lucide-react";
+import { useTranslation } from "@plane/i18n";
 import { Spinner, Tooltip } from "@plane/ui";
 import { useTranslation } from "@plane/i18n";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useTranslation'. Consider to delete it or rename it.

'useTranslation' is defined here:


(lint/suspicious/noRedeclare)